src/lib/annotations/image/ImageAnnotator.js

```diff
@@ -85,6 +85,8 @@ class ImageAnnotator extends Annotator {
      * @return {AnnotationThread} Created annotation thread
      */
     createAnnotationThread(annotations, location, type) {
+        this.emit('annotationmodeexit');
```

Comment:
Can you emit what kind of mode was exited? I'm wondering if developers may want to hide/show different UI depending on what annotation mode is left/entered